coordlib.py

```diff
@@ -1,7 +1,7 @@
 def get_decimal_from_dms(dms, ref):
-    degrees = dms[0][0] / dms[0][1]
-    minutes = dms[1][0] / dms[1][1] / 60.0
-    seconds = dms[2][0] / dms[2][1] / 3600.0
+    degrees = dms[0]
+    minutes = dms[1] / 60.0
+    seconds = dms[2] / 3600.0
```

Comment:
do these lines truly give out correct result?

Reply:
yea
i think this is because api of Pillow 8 changed

Reply:
sweet